### DISCUSSION TITLE
Lines translated in Weblate do not appear in Flarum

### DISCUSSION THREAD

**Discussion by DursunCanPoyraz** [rob006-software/flarum-translations#569] [General]
Lines of this extension are not visible.

https://weblate.rob006.net/projects/flarum/v17development-seo/tr/

![image](https://user-images.githubusercontent.com/74768208/103148775-425b7300-4774-11eb-83d5-e55f7ce7f6c5.png)

**Comment by rob006** [comment]
See note at https://github.com/rob006-software/flarum-translations/wiki#how-long-does-it-take-for-the-text-translated-in-weblate-to-appear-in-the-language-pack - language pack maintainer needs to release new version with new translations, or you should use branch as constraint in `composer.json` instead of stable releases.
